code/obj/machinery/computer.dm

```diff
@@ -12,19 +12,77 @@
 	/// does it have a glow in the dark screen? see computer_screens.dmi
 	var/glow_in_dark_screen = TRUE
 	var/image/screen_image
+
+	var/can_reconnect = 0 //Set to 1 to make multitools call connection_scan. For consoles with associated equipment (cloner, genetek etc)
+	var/obj/item/circuitboard/circuit_type = null
+	Topic(href, href_list)
+		if (..(href, href_list))
+			return 1
+		playsound(src.loc, 'sound/machines/keypress.ogg', 30, 1, -15)
+
+	attack_hand(var/mob/user)
+		..()
+		if (!user.literate)
+			boutput(user, "<span class='alert'>You don't know how to read or write, operating a computer isn't going to work!</span>")
+			return 1
+		interact_particle(user,src)
```

Comment:
Return value is possibly different? Previously `. = ..()`. Unsure if it matters, but possible it does. Would check what the parent's `attack_hand` returns, if anything (if nothing, fine to do as you have it).

Reply:
Adjusted back to `. = ..()`

Reply:
I don't think it did anything, but just to be sure. Parents called parents all the way up to obj.